GitHubUserSearch/GitHubUserSearch/AppDelegate.swift

```diff
@@ -16,31 +16,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
    lazy var model = Model()
```

Comment:
Maybe it's better to make a singleton class, something like Model provider that will contain logic of model creation and fetching from any place of code?

Reply:
I have thought about it, but you never can tell who is the senior developer - a Singletone hater or not.

anyway. the AppDelegate is a singletone, so for the test project to work this should be onough